src/leap/mail/mail.py

```diff
@@ -190,11 +212,15 @@ def get_subpart(self, part):
 
         sub_pmap = self._pmap.get("part_map", {})
         try:
-            part_map = sub_pmap[str(part + 1)]
+            part_map = sub_pmap[str(part)]
         except KeyError:
-            logger.debug("getSubpart for %s: KeyError" % (part,))
+            log.msg("getSubpart for %s: KeyError" % (part,))
```

Comment:
to ease debug, I run mail offline with twistd.
it's amazing to see how quickly a pure-reactor app behaves (mainly launching and shutting down)